authentication/authentication-internal-service/src/main/resources/bootstrap.properties

```diff
@@ -49,3 +49,18 @@ openapi.service.servers[0].description=Id Authentication Internal Service
 openapi.group.name=${openapi.info.title}
 openapi.group.paths[0]=/**
 springdoc.swagger-ui.disable-swagger-default-url=true
+

```

Comment:
remove the ctrl-M characters from this file.

Reply:
Not able to see this ctrl-M character either in local or directly in github https://github.com/mosip/id-authentication/blob/a90ddf10de99ff23c2a494192d39211c907a125b/authentication/authentication-internal-service/src/main/resources/bootstrap.properties

https://github.com/mosip/id-authentication/pull/1346/files Raised 1 sample PR there also it's showing.